packages/runtime/src/next/client.ts

```diff
@@ -636,6 +675,10 @@ export class Makeswift {
     return result.table
   }
 
+  /**
+   * @deprecated `getSitemap` is deprecated. We recommend constructing a sitemap
+   * using data from `getPages` instead.
+   */
```

Comment:
I wonder if we should go ahead and remove this method 🤔 

I know we've talked about deprecating it but since there's feature parity with the new `getPages` and it's a new minor, it's probably ok to remove it?

Reply:
Had an offline discussion, we're going to deprecate the method for now and remove it in a future release